tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1023,9 +1067,217 @@ def fit(
             calibrate_decision_threshold=calibrate_decision_threshold,
             infer_limit=infer_limit,
         )
-        self.save()
+        if dynamic_stacking:
+            logger.info(
```

Comment:
Update to `logger.log(20, "Dynamic ...`

Puts it higher in logging level so it shows up by default.

Reply:
```f"Dynamic stacking is enabled (dynamic_stacking={dynamic_stacking}). AutoGluon will ...```